README.md

```diff
@@ -1,3 +1,8 @@
+[![CircleCI](https://circleci.com/gh/ministryofjustice/community-api/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/community-api)
+[![Docker Repository on Quay](https://quay.io/repository/hmpps/community-api/status "Docker Repository on Quay")](https://quay.io/repository/hmpps/community-api)
+[![Runbook](https://img.shields.io/badge/runbook-view-172B4D.svg?logo=confluence)](https://dsdmoj.atlassian.net/wiki/spaces/NOM/pages/1739325587/DPS+Runbook#Deploying-Community-API)
+[![API docs](https://img.shields.io/badge/API_docs_(needs_VPN)-view-85EA2D.svg?logo=swagger)](https://community-api.test.delius.probation.hmpps.dsd.io/swagger-ui.html)
```

Comment:
You've linked to the non vpn version of the api docs, but badged it as the needs VPN version?

Reply:
Cool thanks, Andy is going to fix in next PR.